File: 0344-reverse-string/0344-reverse-string.ts
/**
 Do not return anything, modify s in-place instead.
 */
function reverseString(s: string[]): void {
  const copy = [...s];
  let curr = s.length - 1;
  while (curr >= 0) {
    s[curr] = copy[s.length - curr - 1];
    curr -= 1;
  }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        copy = list(s)
        i = 0
        while i < len(s):
            s[i] = copy[len(s) - i - 1]
            i += 1

        